src/include/dng/hts/vcf.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2015 Reed A. Cartwright
+ * Authors:  kdai1 <kdai1@asu.edu>
+ *
+ * This file is part of DeNovoGear.
+ *
+ * DeNovoGear is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#ifndef CXX_HTS_VCF_H
+#define CXX_HTS_VCF_H
+
+#include "dng/hts/hts.h"
+#include "dng/matrix.h"
+
```

Comment:
I want to keep the dng/hts wrappers as self contained as possible
```
#include "hts.h"
```
What are you using out of dng/matrix.h?  If it is for DNG specific features, we need to create a a bcf.h header in the dng library and move that code to there.